scalableminds/webknossos
# Jump to the center of the bounding box if find-data doesn't find data

### URL of deployed dev instance (used for testing):
- https://jumptolayercenter.webknossos.xyz

### Steps to test:
- open a dataset for which find-my-data fails and open use the find-my-data feature. it should jump to the middle of the layer's bbox.

### Issues:
- fixes #4287
- fixes #4322 (tested by throwing an error in the save saga and ensuring that the navbar turns red)

------
